src/engraving/dom/breath.cpp

```diff
@@ -183,6 +183,15 @@ String Breath::accessibleInfo() const
     return SymNames::translatedUserNameForSymId(m_symId);
 }
 
+//---------------------------------------------------------
+//   translatedSubtypeUserName
+//---------------------------------------------------------
+
+String Breath::translatedSubtypeUserName() const
```

Comment:
Wouldn't it be better to implement `subtypeUserName` rather than only `translatedSubtypeUserName`?